GeneralsMD/Code/GameEngine/Source/GameClient/MessageStream/CommandXlat.cpp

```diff
@@ -2881,6 +2881,14 @@ GameMessageDisposition CommandTranslator::translateGameMessage(const GameMessage
 
 		}	// end select previous worker
 
+		//-----------------------------------------------------------------------------------------
+		case GameMessage::MSG_META_SELECT_NEXT_IDLE_WORKER:
+		{
+			if (TheInGameUI)
+				TheInGameUI->selectNextIdleWorker();
+			break;
```

Comment:
Looks like `disp = DESTROY_MESSAGE;` is missing? I think it marks the message as consumed?